osfoffline/filesystem_manager/osf_event_handler.py

```diff
@@ -204,6 +204,7 @@ def on_modified(self, event):
         """
 
         if isinstance(event, DirModifiedEvent):
+            # AlertHandler.warn('Modifying folders is not recommended')
```

Comment:
This will run every time any change is made to any folder.  We don't want this.  I'll push up a commit without it if you want.  TBH totally forgot about this change.